core/common/src/main/java/alluxio/AbstractClient.java

```diff
@@ -173,6 +179,16 @@ public synchronized void connect() throws IOException, ConnectionFailedException
         afterConnect();
         checkVersion(getClient(), getServiceVersion());
         return;
+      } catch (IOException e) {
+        if (FRAME_SIZE_EXCEPTION_PATTERN.matcher(e.toString()).find()) {
+          // See an error like "Frame size (67108864) larger than max length (16777216)!",
+          // pointing to the helper page.
+          LOG.error("Failed to set up a connection to " + getServiceName() + " "
+              + mMode + " @ " + mAddress + " due to the mismatched transport. Please refer to "
+              + RuntimeConstants.ALLUXIO_WEB_DOC + "/en/Debugging-Guide.html "
```

Comment:
- We don't have a page in `docs/` that can stay across versions.
- for version < 1.3-SNAPSHOT, the version specific docs will not have it. but later will have the FAQ in their `docs/` when version >= 1.3-SNAPSHOT
- Alternatively, we could patch `docs/` for earlier versions if necessary